src/components/PoolingWidget/index.tsx

```diff
@@ -182,10 +184,15 @@ const PoolingInterface: React.FC = () => {
   // Avoid displaying an empty list of tokens when the wallet is not connected
   const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
 
-  // TODO: switched to tagged tokens @anxo @leandro
-  const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId).filter(({ symbol }) => symbol !== 'WETH'), [
-    fallBackNetworkId,
-  ])
+  const tokens = useMemo(() => {
+    return (
+      // Get all the tokens for the current network
+      tokenListApi
+        .getTokens(fallBackNetworkId)
+        // Filter out the tokens not in the list
+        .filter(({ symbol }) => symbol && LIQUIDITY_TOKEN_LIST.has(symbol))
```

Comment:
```suggestion
        .filter(({ symbol }) => LIQUIDITY_TOKEN_LIST.has(symbol))
```

Checking for `undefined` in the set should work, right?

Reply:
I'm afraid it's expecting a String:
![image](https://user-images.githubusercontent.com/2352112/73868480-ce0c9e00-4848-11ea-8229-cb6ddf930b40.png)

Reply:
Ah right, typescript, typescripting :)